src-electron/db/query-config.js

```diff
@@ -20,1160 +20,1202 @@
  *
  * @module DB API: user configuration queries against the database.
  */
-const dbApi = require('./db-api.js')
```

Comment:
The whole file seem to have changed and makes it very difficult to actually review the changes.
Can you fix this?

Reply:
Fixed

Reply:
> <img alt="Screen Shot 2022-08-26 at 11 26 58 AM" width="304" src="https://user-images.githubusercontent.com/61332088/186939742-40673aba-9b1e-4317-b487-6c550b70a415.png">
> 
> During the call, I believe the agreed intention is to remove all the text (Edit/Copy/etc) and only leave icons. The tooltips can include the texts. For the expand / un-expand, let's use the default up / down arrow
> 
> e.g. <img alt="Screen Shot 2022-08-26 at 11 28 43 AM" width="339" src="https://user-images.githubusercontent.com/61332088/186940072-006d1db3-6243-4128-849e-20649ea21660.png">

I am still the same issue for the UI.